apps/antalmanac/src/components/ScheduleManagement/ScheduleManagementTabsContent.tsx

```diff
@@ -0,0 +1,50 @@
+import { lazy, Suspense } from 'react';
+
+import darkModeLoadingGif from '../RightPane/CoursePane/SearchForm/Gifs/dark-loading.gif';
+import loadingGif from '../RightPane/CoursePane/SearchForm/Gifs/loading.gif';
+
+import { ScheduleCalendar } from '$components/Calendar/CalendarRoot';
+import { AddedCoursePane } from '$components/RightPane/AddedCourses/AddedCoursePane';
+import { CoursePaneRoot } from '$components/RightPane/CoursePane/CoursePaneRoot';
+import { useThemeStore } from '$stores/SettingsStore';
+import { useTabStore } from '$stores/TabStore';
+
+
+const UCIMap = lazy(() => import('../Map/Map'));
+
+export function ScheduleManagementTabsContent() {
```

Comment:
Can you add a comment (or even better yet, rename things) such that the distinction between this and `ScheduleManagementTabs`? It took me way too long to see that.